frontend/src/pages/modelServing/screens/projects/utils.ts

```diff
@@ -629,8 +627,17 @@ export interface ModelInfo {
   updatedDate: string;
 }
 
-export const fetchNIMModelNames = async (): Promise<ModelInfo[] | undefined> => {
-  const configMap = await getNIMResource<ConfigMapKind>(NIM_CONFIGMAP_NAME);
+export const fetchNIMModelNames = async (
+  dashboardNamespace: string,
```

Comment:
In contrast of my other point -- this is needed... functions don't have access to hooks. This is needed 👍